undebt/pattern/interface.py

```diff
@@ -74,10 +76,35 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        return _load_module(path)
+
     pattern_name = os.path.splitext(os.path.basename(path))[0]
     return imp.load_source(pattern_name, path)
 
 
+def _load_module(full_name):
+    try:
+        return importlib.import_module(full_name)
+    except ImportError as e:
+        raise e
+    finally:
+        # Unshim path
+        sys.path = sys.path[1:]
```

Comment:
The `finally` should ensure that `sys.path` returns to its original state before the method exits.

I'm thinking about your first point. Let's say I'm in `~/foo`, and I have undebt installed on my system, and `~/foo/bar/baz.py` and `~/foo/bar/__init__.py` exist. If I then do `undebt -p bar.baz`, will the current directory (that is, `~/foo`) be included in the list of paths that `__import__` will search?